server/tables/tables.go

```diff
@@ -672,24 +712,47 @@ func PostAutoMigrate(db *gorm.DB) error {
 		}
 	}
 
-	// Drop old columns at the very end of the migration.
-	for _, ref := range []struct {
+	type ColRef struct {
 		table  Table
 		column string
-	}{
+	}
+
+	colsToDelete := []ColRef{
 		// Group.api_key has been migrated to a single row in the APIKeys table.
 		{&Group{}, "api_key"},
-	} {
+	}
+
+	if *dropInvocationPKCol {
```

Comment:
Can you add a comment here to document why this is hidden behind a flag?

Reply:
Done